tests/rust_syscall_tests/src/get_att.rs

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#![no_std]
+#![no_main]
+#![feature(naked_functions, asm_sym)]
+
+rust_syscall_tests::startup!();
+
+use rust_syscall_tests::*;
+
+fn main() -> Result<()> {
+    if !is_enarx() {
```

Comment:
We had a `nil` backend, which would run the `exec` directly. Not sure, if we want to reintroduce this ever. Maybe it can be removed.. for the sake of completeness, I just left it in.

Btw, you can run the produced test binaries just fine without enarx. Could make a blogpost about the smallest rust hello world binary :)

Reply:
Heh, it would be nice to have a more modern blog post than http://mainisusuallyafunction.blogspot.com/2015/01/151-byte-static-linux-binary-in-rust.html .